tests/test_bulk_rx_traffic.py

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env python
 # Copyright 2016-2021 XMOS LIMITED.
 # This Software is subject to the terms of the XMOS Public Licence: Version 1.
-import xmostest
+import Pyxsim
```

Comment:
On first inspection not quite sure why all the tests need to import Pyxsim here..

Reply:
You are right. Removed from every test.py